py/emitinlinerv32.c

```diff
@@ -196,7 +196,6 @@ typedef enum {
     CALL_R,   // Opcode Register
     CALL_RL,  // Opcode Register, Label
     CALL_N,   // Opcode
-    CALL_I,   // Opcode Immediate
```

Comment:
You forgot to remove the associated typedef on line 219 (`typedef void (*call_i_t)(asm_rv32_t *state, mp_int_t immediate);`).